packages/client/src/AdminPostCreation.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+
+// Data looks like this (for backend)
+// console.log({
+//     title:'string',
+//     body:'string',
+//     isDraft:boolean
+// })
+let backend = process.env.REACT_APP_BACKEND;
+let endpoint = `${backend}/api/createPost`;
```

Comment:
These should be `const`

Reply:
Resolved these, changed to "const" and committed here on GitHub :)